src/modules/getLikes.js

```diff
@@ -0,0 +1,17 @@
+export const invAPI = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/nGXJb8jNMlAqNOxXKCnW/likes';
```

Comment:
The project uses the Involvement API to record the different user interactions (likes and comments) 📝